tests/ops/service/connectors/fides/test_fides_client.py

```diff
@@ -126,7 +125,8 @@ def test_authenticated_request_parameters(
             request.url
             == test_fides_client.uri + "/testpath?param1=value1&param2=value2"
         )
-        assert request.body == "key1=value1&key2=value2"
+        # request.read()
```

Comment:
@adamsachs FYI it looks like tuples are no longer supported in `httpx` for the `data` param (it breaks the `read()` command)

Do you know if we're using that feature anywhere else? Any opposition to removing this test given that fact?

Reply:
i think we're fine to remove this test, i don't think we were actually passing tuples anywhere, this was just t obe comprehensive